modules/EnsEMBL/Web/Query/Availability/Gene.pm

```diff
@@ -41,15 +41,15 @@ sub _counts {
   # but metazoa have three clustersets
   for my $clusterset_id (@$clusterset_ids) {
     warn "CLUSTERSET ID " . $clusterset_id;
-    warn $compara_member->number_of_paralogues($clusterset_id);
+    warn $compara_member->number_of_paralogues($clusterset_id) if defined $compara_member;
```

Comment:
these two warning lines must be the remnants of my clumsy debugging efforts; let's remove them

Reply:
Oh right. OK, will do!